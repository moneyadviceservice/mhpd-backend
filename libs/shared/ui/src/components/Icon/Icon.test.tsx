import React from 'react';
import { render, screen } from '@testing-library/react';
import { Icon, IconType } from '.';

describe('Icon component', () => {
  it('renders correctly', () => {
    render(
      <Icon
        data-testid="icon"
        className="test-class"
        type={IconType.CHEVRON}
      />,
    );
    const icon = screen.getByTestId('icon');
    expect(icon).toMatchSnapshot();
  });
});
