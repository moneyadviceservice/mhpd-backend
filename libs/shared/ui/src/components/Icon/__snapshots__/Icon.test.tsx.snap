// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component renders correctly 1`] = `
<svgr-mock
  classname="test-class"
  data-testid="icon"
/>
`;
